apps/portlet-explorer/src/main/webapp/javascript/eXo/ecm/MultiUpload.js

```diff
@@ -283,7 +283,7 @@
 	function cleanName(oldName) {
 	
 		if (!oldName || oldName==null ||oldName==undefined) return oldName;
-		specialChar = "[]/'\":;";
+		specialChar = "[]'\":;";
```

Comment:
As explained yesterday, you cannot remove a special character like this. It is a major risk to create side effects

Reply:
ok, Romain .I left it as is and used the function encodeURIComponent to encode the special characters in URI which be affected in param "currentFolder".